eth/main.cpp

```diff
@@ -1441,8 +1441,10 @@ int main(int argc, char** argv)
 			}
 		}
 #if ETH_EVMJIT
-		else if (arg == "-J" || arg == "--jit")
-			jit = true;
+		else if (arg == "--vm=jit")
```

Comment:
should be `"--vm" "jit"` for consistency with everything else that takes a param e.g. `"--verbosity" "4"`